app-typescript/components/TreeSelect/TreeSelect.tsx

```diff
@@ -43,6 +52,8 @@ interface IPropsBase<T> extends IInputWrapper {
     placeholder?: string;
     searchPlaceholder?: string;
     zIndex?: number;
+    dragAndDrop?: boolean;
```

Comment:
replace `dragAndDrop` and `onDrag` props with `sortable: boolean`

Reply:
so we don't need a function onDrag?

Reply:
No. Do you see a reason why would we need it?